Promise.js

```diff
@@ -126,7 +108,7 @@
 	};
 
 	Promise.all = function () {
-		var args = Array.prototype.slice.call(arguments.length === 1 && isArray(arguments[0]) ? arguments[0] : arguments);
+		var args = Array.prototype.slice.call(arguments.length === 1 && Array.isArray(arguments[0]) ? arguments[0] : arguments);
```

Comment:
This behavior is not spec-compliant. all should reject on non-iterable arguments.

(I would file an issue but issues are disabled.)